quartz/plugins/emitters/folderPage.tsx

```diff
@@ -38,12 +38,21 @@ export const FolderPage: QuartzEmitterPlugin<Partial<FullPageLayout>> = (userOpt
     getQuartzComponents() {
       return [Head, Header, Body, ...header, ...beforeBody, pageBody, ...left, ...right, Footer]
     },
-    async getDependencyGraph(_ctx, _content, _resources) {
+    async getDependencyGraph(_ctx, content, _resources) {
       // Example graph:
-      // nested/file.md --> nested/file.html
-      //          \-------> nested/index.html
-      // TODO implement
-      return new DepGraph<FilePath>()
+      // nested/file.md --> nested/index.html
+      // nested/file2.md ------^
+      const graph = new DepGraph<FilePath>()
+
+      content.map(([_tree, vfile]) => {
+        const slug = vfile.data.slug
+        const folderName = path.dirname(slug ?? "") as SimpleSlug
+        if (slug && folderName !== "." && folderName !== "tags") {
+          graph.addEdge(vfile.data.filePath!, joinSegments(folderName, "index.html") as FilePath)
```

Comment:
Hmm btw quartz does cleanup url, so I wonder should we just do `folder/` as slug instead?

I guess do need the actual html file here?

Reply:
We could do folder/ instead of folder/index.html, but it's useful to see which file exactly is the result of this emitter.

Also, this code is mostly copied from the emit method further down.

Reply:
iirc below it only check for `folderName/index` since it cleans the URL, and then the folderPage actually do a redirect to `folderName`.

I think for the edge `folderName/index` should work.